### PR TITLE
Fix #67: Add QSS._prefetch_related_lookups

### LINKED ISSUE #67: AttributeError: 'QuerySetSequence' object has no attribute '_prefetch_related_lookups'
Any idea why ModelChoiceIterator does not work with QSS anymore ? Will look further into this anyway, apparently it broke django-autocomplete-light

```
self = <django.forms.models.ModelChoiceIterator object at 0x7f801689ced0>

    def __iter__(self):

        if self.field.empty_label is not None:

            yield ("", self.field.empty_label)

        queryset = self.queryset

        # Can't use iterator() when queryset uses prefetch_related()

>       if not queryset._prefetch_related_lookups:

E       AttributeError: 'QuerySetSequence' object has no attribute '_prefetch_related_lookups'

../.tox/base-py37-django21-sqlite/lib/python3.7/site-packages/django/forms/models.py:1137: AttributeError
```

### PR DESCRIPTION
**Pull request by jpic**
Sorry for the delay...

### CODE REVIEW

**Comment by dicato** [comment]
Looks good to me! I'll let @clokep weigh in as you both discussed this approach in the issue.

File: tests/test_querysetsequence.py [hunk 1]
```diff
@@ -340,6 +340,14 @@ def test_prefetch_related(self):
             authors = [b.author.id for b in books]
         self.assertEqual(authors, self.EXPECTED_ORDER)
 
+    def test_prefetch_related_lookups(self):
```
Comment (clokep) [inline tests/test_querysetsequence.py:343]: Please add a docstring about what this is testing.

File: queryset_sequence/__init__.py [hunk 2]
```diff
@@ -331,6 +331,14 @@ def _fetch_all(self):
         if self._result_cache is None:
             self._result_cache = list(self._iterable_class(self))
 
+    @property
+    def _prefetch_related_lookups(self):
+        # for django ModelChoiceField, pending for removal
+        return any(map(
+            lambda qs: qs._prefetch_related_lookups,
+            self._querysets
+        ))
```
Comment (clokep) [inline queryset_sequence/__init__.py:340]: This doesn't look right, `_prefetch_related_lookups` seems to be a tuple on `QuerySet`, not a boolean. If the goal is to just fix the `ModelChoiceField` but then the comment above this needs to be expanded.

Either way I'd like to see the comment describe what versions of Django this affects (and a reference to the Django ticket that will fix this).

File: tests/test_querysetsequence.py (hunk 1 above)
Comment (clokep) [reply tests/test_querysetsequence.py:343]: Would it make more sense to create the `ModelChoiceField` and call the API on it to ensure it "works" as expected?

Comment (clokep) [reply tests/test_querysetsequence.py]: I added this test as suggested.

File: queryset_sequence/__init__.py (hunk 2 above)
Comment (clokep) [reply queryset_sequence/__init__.py]: I updated the comment here and return a tuple, which...is still funky, but at least is the same type.

**Comment by clokep** [comment]
@jpic I pushed a couple of commits, let me know if you think the changes look good!

**Comment by clokep** [comment]
My plan is to merge this and do a release next week.

**Pull request merged**